lib/airbrake/railtie.rb

```diff
@@ -12,13 +12,16 @@ class Railtie < ::Rails::Railtie
 
     initializer "airbrake.middleware" do |app|
 
-      middleware = if defined?(ActionDispatch::DebugExceptions)
-        # Rails >= 3.2.0
-        "ActionDispatch::DebugExceptions"
-      else
-        # Rails < 3.2.0
-        "ActionDispatch::ShowExceptions"
-      end
+      middleware =
+        if !::Rails.env.production? && defined?(::BetterErrors::Middleware)
```

Comment:
The only good reason is to make sure we don't break apps in production. Better errors already check if env is different from production before loading their middleware, but I just wanted additional check in our gem.